curl/README.md

```diff
@@ -0,0 +1,68 @@
+
+# `curl`
+
+This task performs curl operation to transfer data from internet .
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/curl/curl.yaml
+```
+
+## Parameters
+
+* **urlimages/curl**: `curl` CLI container image to run this task.
+
+* **options**: The arguments to pass to `curl` CLI.  _default_: `[""]`
+* **url**: The url we want to download file from.  _default_: `""`
+## Usage
+
+### Running the Task
+
+#### Viewing request headers and connection details
+
+You can invoke `curl -v url` to get request header from specified url:
+
+Create a file, `curl-header-run.yaml`:
+
+```
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: curl-header-run
+spec:
+  params:
+    - name: url
+      value: "www.google.com"
+    - name: options
+      value:
+       - "-v"
+  taskRef:
+    name: curl
+```
+
+Run it with `kubectl create -f curl-header-run.yaml`
```

Comment:
same as https://github.com/tektoncd/catalog/pull/357#discussion_r437691470

either use `generateName` or `kubectl apply`

Reply:
Hmm, also `curl -v` will print out both headers as well as the body. For only headers, we can use `curl -i`